### ISSUE TITLE
scripts/setup-intellij.sh not working

### ISSUE THREAD

**Issue by abergmeier** [bazelbuild/bazel#1493]
With latest successful test build, IntelliJ setup does not work.
Perhaps it would be good to add it to the test.

**Comment by janakdr** [comment]
Looks like this is because of Windows:
```
ERROR: /usr/local/google/home/janakr/bazel/src/main/native/BUILD:66:1: Executing genrule //src/main/native:windows_jni failed: namespace-sandbox failed: error executing command /usr/local/google/home/janakr/.cache/bazel/_bazel_janakr/6264bddbcc0465383fcc9614dbc9d23a/execroot/bazel/_bin/namespace-sandbox ... (remaining 5 argument(s) skipped).
Visual Studio not found
ls: cannot access C:/Program Files (x86): No such file or directory
```

I've mailed out a [fix](https://bazel-review.googlesource.com/#/c/4020). Adding a test would be a good idea, but my understanding is that this script may not be needed for much longer, if we can add first-class IntelliJ support to Bazel, just as we've recently done with Eclipse.

**Comment by kchodorow** [comment]
Fixed by ca8db60193b93968902a5c25f3786a8d0e71268c.

**Comment by abergmeier-dsfishlabs** [comment]
Thanks @janakdr, you saved me from the clutches of Eclipse :)
